corehq/form_processor/abstract_models.py

```diff
@@ -124,10 +124,7 @@ def get_sync_token(self):
             try:
                 return get_properly_wrapped_sync_log(self.last_sync_token)
             except ResourceNotFound:
-                logging.exception('No sync token with ID {} found. Form is {} in domain {}'.format(
-                    self.last_sync_token, self.form_id, self.domain,
-                ))
-                raise
+                pass
```

Comment:
it looks like this now returns None rather than failing hard. just confirming that the code which uses this can handle the None return value?